lib/src/reactive_widget.dart

```diff
@@ -5,32 +5,30 @@ import 'reactive_base_widget.dart';
 
 class ReactiveWidget<T> extends ReactiveBaseWidget<T> {
   final RxWidget<T> widget;
-  final RxErrorWidget<T> errorWidget;
-  final Widget placeHolderWidget;
+  final RxErrorWidget<T>? errorWidget;
+  final Widget? placeHolderWidget;
 
   const ReactiveWidget({
-    Key key,
-    @required Stream<T> stream,
-    T initialData,
-    @required this.widget,
+    Key? key,
+    required Stream<T> stream,
+    T? initialData,
+    required this.widget,
     this.placeHolderWidget,
     this.errorWidget,
-  })  : assert(stream != null),
-        assert(widget != null),
-        super(stream, initialData, key: key);
+  }) : super(stream, initialData, key: key);
 
   @override
   Widget build(BuildContext context, T data) => widget(data);
 
   @override
   Widget placeHolderBuild(BuildContext context) {
-    if (placeHolderWidget != null) return placeHolderWidget;
+    if (placeHolderWidget != null) return placeHolderWidget!;
     return super.placeHolderBuild(context);
   }
 
   @override
-  Widget errorBuild(BuildContext context, Object error) {
-    if (errorWidget != null) return errorWidget(error);
+  Widget errorBuild(BuildContext context, Object? error) {
```

Comment:
Why is error nullable here? Normally the ErrorBuilder should never be called if there is no error or do I miss something here?